pom.xml

```diff
@@ -16,6 +16,7 @@
         <maven.checkstyle.plugin.configLocation>
             https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
+        <kotlin.version>2.0.0</kotlin.version>
```

Comment:
Why do you need change pom file and Kotlin also?

Reply:
I did not do it